code/engine/xrRenderCommon/NvTriStripObjects.h

```diff
@@ -65,7 +65,8 @@ class NvEdgeInfo {
     // ref and unref
     void Unref() {
         if (--m_refCount == 0) {
-            Memory.mem_free(this);
+            NvEdgeInfo* self = dynamic_cast<NvEdgeInfo*>(this);
```

Comment:
What the purpose of dynamic_cast?

Reply:
it will be removed later with replacement of xr_delete